Source/MSBuildProjectBuilder/Assets/Scenes/SampleScene.unity

```diff
@@ -1,259 +0,0 @@
-%YAML 1.1
```

Comment:
>YAML [](start = 1, length = 4)

Did you intend to fully delete the sample scene? I have been using this for testing.

Reply:
Oh! Sorry, I was looking to cleanup, and saw the scene being empty, I was presumptious will undo.

Reply:
Erm, if it's empty then I missed something on a previous checkin. So yea, I guess leave it there and I'll fix it up.

---
In reply to: [314902035](https://github.com/microsoft/MSBuildForUnity/pull/11#discussion_r314902035) [](ancestors = 314902035)